sessionctx/variable/session.go

```diff
@@ -646,6 +646,9 @@ type SessionVars struct {
 	// allowInSubqToJoinAndAgg can be set to false to forbid rewriting the semi join to inner join with agg.
 	allowInSubqToJoinAndAgg bool
 
+	// allowPreferRangeScan allows optimizer to always prefer range scan over table scan.
+	allowPreferRangeScan bool
```

Comment:
how about this, which is shorter:
```suggestion
	preferRangeScan bool
```

Reply:
it's ok